### PR TITLE
Fix: PlatformIO build

### LINKED ISSUE #6: PlatformIO Build Fails
PlatformIO configuration is not setup for the correct board. Looks like this file was missed when setting up the Marlin branch.

### PR DESCRIPTION
**Pull request by mikejr83**
Fix `platformio.ini` for correct build environment.

Closes #6

### CODE REVIEW

**Comment by attacktwinkie** [comment]
What version of Platformio are you basing this on?

File: platformio.ini [hunk 1]
```diff
@@ -588,12 +615,12 @@ board         = esp32dev
...
-upload_port   = /dev/ttyUSB0
-lib_deps      =
+upload_port = /dev/ttyUSB0
+lib_deps =
   AsyncTCP=https://github.com/me-no-dev/AsyncTCP/archive/master.zip
   ESPAsyncWebServer=https://github.com/me-no-dev/ESPAsyncWebServer/archive/master.zip
-lib_ignore    = LiquidCrystal, LiquidTWI2, SailfishLCD, SailfishRGB_LED
-src_filter    = ${common.default_src_filter} +<src/HAL/HAL_ESP32>
+lib_ignore  = LiquidCrystal, LiquidTWI2, SailfishLCD, SailfishRGB_LED
+src_filter  = ${common.default_src_filter} +<src/HAL/HAL_ESP32>
```
Comment (attacktwinkie) [inline platformio.ini:623]: Remove the Whitespace changes.

File: platformio.ini [hunk 2]
```diff
@@ -354,16 +354,16 @@ monitor_speed = 250000
...
+build_flags =
+  ${common.build_flags} -DUSBCON -DUSBD_VID=0x0483 '-DUSB_MANUFACTURER="Unknown"' '-DUSB_PRODUCT="ARMED_V1"' -DUSBD_USE_CDC
   -O2 -ffreestanding -fsigned-char -fno-move-loop-invariants -fno-strict-aliasing -std=gnu11 -std=gnu++11
   -IMarlin/src/HAL/HAL_STM32
-lib_deps      = ${common.lib_deps}
-lib_ignore    = Adafruit NeoPixel, SoftwareSerial
-src_filter    = ${common.default_src_filter} +<src/HAL/HAL_STM32>
+lib_deps    = ${common.lib_deps}
+lib_ignore  = Adafruit NeoPixel, SoftwareSerial
+src_filter  = ${common.default_src_filter} +<src/HAL/HAL_STM32>
```
Comment (attacktwinkie) [inline platformio.ini:366]: These are all whitespace changes that should be removed.

File: platformio.ini [hunk 3]
```diff
@@ -18,7 +18,7 @@
 [platformio]
 src_dir      = Marlin
 boards_dir   = buildroot/share/PlatformIO/boards
-default_envs = megaatmega2560
+default_envs = mks_robin_pro
```
Comment (attacktwinkie) [inline platformio.ini:21]: Approved

File: platformio.ini [hunk 4]
```diff
@@ -412,7 +412,20 @@ build_unflags = -std=gnu++11
...
+platform      = ststm32
+framework     = arduino
+board         = genericSTM32F103RC
+extra_scripts = buildroot/share/PlatformIO/scripts/mks_robin_lite3.py
+build_flags   = !python Marlin/src/HAL/HAL_STM32F1/build_flags.py
+  ${common.build_flags} -std=gnu++14
+build_unflags = -std=gnu++11
+src_filter    = ${common.default_src_filter} +<src/HAL/HAL_STM32F1>
+lib_deps      = ${common.lib_deps}
+lib_ignore    = Adafruit NeoPixel, SPI
```
Comment (attacktwinkie) [inline platformio.ini:428]: This is great, but you need to add one line of whitespace above it.

File: platformio.ini [hunk 5]
```diff
@@ -443,6 +456,20 @@ src_filter    = ${common.default_src_filter} +<src/HAL/HAL_STM32F1>
...
+platform      = ststm32@<6.2.0
+framework     = arduino
+board         = genericSTM32F103ZE
+extra_scripts = buildroot/share/PlatformIO/scripts/mks_robin_pro.py
+build_flags   = !python Marlin/src/HAL/HAL_STM32F1/build_flags.py
+  ${common.build_flags} -std=gnu++14 -DSTM32_XL_DENSITY
+build_unflags = -std=gnu++11
+src_filter    = ${common.default_src_filter} +<src/HAL/HAL_STM32F1>
+lib_deps      = ${common.lib_deps}
+lib_ignore    = Adafruit NeoPixel, SPI
```
Comment (attacktwinkie) [inline platformio.ini:472]: NO changes needed, this is pefect.

**Review by attacktwinkie — CHANGES_REQUESTED** [review]
Clean up the whitespace and we'll merge it.

**Comment by attacktwinkie** [comment]
This should be fixed now starting with 2021.2 branch.

**Comment by mikejr83** [comment]
Closing as obsolete.

**Pull request closed without merging**